### PR TITLE
docs: README/Changelog for 0.3.0 release

### PR DESCRIPTION
**Pull request by pierceboggan**

### CODE REVIEW

**Comment by pierceboggan** [comment]
@wenyt For search integration, what do we support searching? Just API name? I want to make sure I'm clear in Changelog / documentation.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 0.3.0
+
+The 0.2.0 release of the Azure API Center extension for Visual Studio Code introduces new functionality to detect breaking changes between two OpenAPI specification documents and improves API discoverability by adding the ability to search for an API in the tree view.
```
Comment (wenytang-ms) [inline CHANGELOG.md:5]: There maybe `0.3.0` ?

File: README.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ Shift-left API design conformance checks into Visual Studio Code with integrated
 
 ![linting](./media/integrated-lint.png)
 
+Ensure that new API versions don't break API consumers with breaking change detection support, powered by Optic.
+
+![image](https://github.com/pierceboggan/vscode-azureapicenter-1/assets/1091304/78777fd4-f7cd-488e-a152-20d0f696fa8a)
```
Comment (wenytang-ms) [inline README.md:15]: now the `api-center` is open source, so please upload the image to `media` folder and reference from it.

**Comment by wenytang-ms** [comment]
> @wenyt For search integration, what do we support searching? Just API name? I want to make sure I'm clear in Changelog / documentation.

Currently search APIs support not only API name, but also with some properties (API kind, created time, lifeCycleStage, description, summary, customProperties and so on).

**Comment by pierceboggan** [comment]
@wenytang-ms - Can you provide some examples of search queries using those properties so I can update the README/docs?

**Comment by wenytang-ms** [comment]
@pierceboggan let me show some sample 
1. if user want to search for all the `rest APIs` in the API Center Instance
rest

2. if user want to search for all the `design` lifecycle APIs, he may 
design

3. if user want to search for all the customProperties like following Apis
API test
testabc

**Comment by pierceboggan** [comment]
@wenytang-ms - AH I see. I know what I was doing wrong. I was doing. "lifecycle:production type:REST"

But from what I see in the screenshots, you don't need to clarify which property you want to filter against?

**Comment by wenytang-ms** [comment]
> @wenytang-ms - AH I see. I know what I was doing wrong. I was doing. "lifecycle:production type:REST"
> 
> But from what I see in the screenshots, you don't need to clarify which property you want to filter against?

You are right, It should be give more clarify in the search box. Let me improve the search experience before the next release, so currently shall we suppose users only to search APIs name?

**Comment by pierceboggan** [comment]
@wenytang-ms That sounds good to me! We don't have to make any changes to the release branch. We can keep as-is for April release, and add the ability to refine search results as described above in the May release.

**Comment by wenytang-ms** [comment]
@pierceboggan Great! I already create a backlog into project to improve the search expericen.
https://github.com/orgs/microsoft/projects/1014?pane=issue&itemId=59249311

**Review by wenytang-ms — APPROVED** [review]

**Review by formulahendry — APPROVED** [review]

**Pull request merged**